src/Gizmos/planeRotationGizmo.ts

```diff
@@ -201,6 +269,100 @@ export class PlaneRotationGizmo extends Gizmo {
         }
     }
 
+    private angleBetween3DCoords(origin: Vector3, coord1: Vector3, coord2: Vector3): number {
```

Comment:
Don't we have anything that can cover this in our math lib? (`GetAngleBetweenVectors`?)

Reply:
Also - 

1) this function, called on each frame, is creating 5 Vectors on each call. This must be avoided. The Temp Vectors are great for such a task.
2) Naming convention